### PR TITLE
Minor README improvements

### PR DESCRIPTION
**Pull request by thbkrkr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ Go to the [releases](https://github.com/elastic/eck-diagnostics/releases) page a
 
 Just execute the binary. On macOS versions with Gatekeeper enabled you have to explicitly allow the execution the first time round, as described in this [support article](https://support.apple.com/en-us/HT202491) (in the section "If you want to open an app that hasn’t been notarized or is from an unidentified developer"). 
 
-By default the tool will run diagnostics for the `elastic-system` namespace, where the ECK operator typically resides, and the `default` namespace.
+To run diagnostics, you need to specify at least the workload resources namespace(s). The operator namespace is set by default to the `elastic-system` namespace, where the ECK operator typically resides.
 
-To run diagnostics, for example, for namespaces `a` and `b` instead:
+Example to run diagnostics for resources namespaces `a` and `b`:
```
Comment (pebrc) [inline README.md:17]:
```suggestion
For example, to run diagnostics for resources namespaces `a` and `b`:
```

File: README.md [hunk 2]
```diff
@@ -71,10 +71,11 @@ In addition to the resources mentioned above, the following Kubernetes resources
...
+* Beat
 * ElasticMapsServer
 * Elasticsearch
 * EnterpriseSearch
 * Kibana
 
 ### Logs
-In the operator namespaces (`-o, --operator-namespaces`) all logs are collected, while in the workload resource namespaces only those logs that originate in Pods managed by ECK are collected .
\ No newline at end of file
+In the operator namespaces (`-o, --operator-namespaces`) all logs are collected, while in the workload resources namespaces only logs from Pods managed by ECK are collected.
```
Comment (pebrc) [inline README.md:81]:
```suggestion
In the operator namespaces (`-o, --operator-namespaces`) all logs are collected, while in the workload resource namespaces only logs from Pods managed by ECK are collected.
```

I don't think we need the plural even though there are ore than one per namespace.

File: README.md [hunk 3]
```diff
@@ -12,14 +12,14 @@ Go to the [releases](https://github.com/elastic/eck-diagnostics/releases) page a
 
 Just execute the binary. On macOS versions with Gatekeeper enabled you have to explicitly allow the execution the first time round, as described in this [support article](https://support.apple.com/en-us/HT202491) (in the section "If you want to open an app that hasn’t been notarized or is from an unidentified developer"). 
 
-By default the tool will run diagnostics for the `elastic-system` namespace, where the ECK operator typically resides, and the `default` namespace.
+To run diagnostics, you need to specify at least the workload resources namespace(s). The operator namespace is set by default to the `elastic-system` namespace, where the ECK operator typically resides.
```
Comment (pebrc) [inline README.md:15]:
```suggestion
To run diagnostics, you need to specify at least the workload resource namespace(s). The operator namespace is set by default to the `elastic-system` namespace, where the ECK operator typically resides.
```

**Review by pebrc — COMMENTED** [review]
cc @alaudazzi

File: README.md [hunk 4]
```diff
@@ -12,14 +12,14 @@ Go to the [releases](https://github.com/elastic/eck-diagnostics/releases) page a
 
 Just execute the binary. On macOS versions with Gatekeeper enabled you have to explicitly allow the execution the first time round, as described in this [support article](https://support.apple.com/en-us/HT202491) (in the section "If you want to open an app that hasn’t been notarized or is from an unidentified developer"). 
 
-By default the tool will run diagnostics for the `elastic-system` namespace, where the ECK operator typically resides, and the `default` namespace.
+To run diagnostics, you need to specify at least the workload resource namespace(s). The operator namespace is set by default to the `elastic-system` namespace, where the ECK operator typically resides.
 
-To run diagnostics, for example, for namespaces `a` and `b` instead:
+For example, to run diagnostics for resources namespaces `a` and `b`:
```
Comment (thbkrkr) [inline README.md:17]:
```suggestion
For example, to run diagnostics for resource namespaces `a` and `b`:
```

**Review by pebrc — APPROVED** [review]

**Pull request merged**